storagemanager/users/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.auth.models import User
+from PIL import Image
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    phone_number = models.CharField(max_length=15)
+    image = models.ImageField(
+        default='default_profile_pic.jpg', upload_to='profile_pics')
+
+    def __str__(self):
+        return f"{self.user.username}'s Profile"
+
+    def save(self, *args, **kwargs):
+        super().save(*args, **kwargs)
+
+        img = Image.open(self.image.path)
+        if img.mode == 'RGBA':
+            img = img.convert('RGB')
+
+        if img.height > 300 or img.width > 300:
+            output_size = (300, 300)
+            img.thumbnail(output_size)
+            img.save(self.image.path)
+
+    @classmethod
+    def find_profile(cls, user_instance):
```

Comment:
I don't think this method should be `Profile` class method. See also comment around its usage.